dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/search/TestSolrClientFactory.java

```diff
@@ -26,31 +22,13 @@
 public class TestSolrClientFactory extends SolrClientFactory {
 
     private static final Logger LOGGER = Logger.getLogger(TestSolrClientFactory.class.getCanonicalName());
-    private static final org.slf4j.Logger SOLR_CONTAINER_LOGGER = org.slf4j.LoggerFactory.getLogger("SolrContainer");
-    
-    private static final GenericContainer<?> SOLR_CONTAINER;
-    
-    static {
-        SOLR_CONTAINER = new GenericContainer<>(new ImageFromDockerfile()
-                    .withFileFromClasspath("Dockerfile", "/docker/test_solr/Dockerfile")
-                    .withFileFromClasspath("schema.xml", "/docker/test_solr/schema.xml")
-                    .withFileFromClasspath("solrconfig.xml", "/docker/test_solr/solrconfig.xml")
-                    .withFileFromClasspath("ror/schema.xml", "/docker/test_solr/ror/schema.xml")
-                    .withFileFromClasspath("ror/solrconfig.xml", "/docker/test_solr/ror/solrconfig.xml")
-                )
-                .withLogConsumer(new Slf4jLogConsumer(SOLR_CONTAINER_LOGGER))
-                .withExposedPorts(8983)
-                .waitingFor(Wait.forHttp("/solr/collection1/select"));
 
-        SOLR_CONTAINER.start();
-    }
-    
     // -------------------- LOGIC --------------------
     
     @Produces
     @Specializes
     public SolrClient produceSolrClient() throws IOException {
-        String urlString = "http://localhost:" + SOLR_CONTAINER.getMappedPort(8983) + "/solr/collection1";
+        String urlString = "http://localhost:8983/solr/collection1";
```

Comment:
I'm thinking if we should use 8983 port. When someone will run integration tests while it has the application running then there is a possibility that we will break user data that he currently have in solr. Maybe changing the port to let's say `8984` would be safer?